recipes/msgpack-c/bld.bat

```diff
@@ -0,0 +1,12 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+    -DMSGPACK_CXX11=YES ^
+    -DCMAKE_BUILD_TYPE=Release ^
```

Comment:
appveyor.yml of msgpack-c uses `-DCMAKE_CXX_FLAGS='"/D_VARIADIC_MAX=10 /EHsc"'`

https://github.com/msgpack/msgpack-c/blob/master/appveyor.yml#L49